e2e/cypress/support/commands.js

```diff
@@ -59,3 +61,104 @@ Cypress.Commands.add('forceVisit', (path, timeout) => {
   cy.visit('/', { timeout })
   cy.window().then(win => win.open(path, '_self'))
 })
+
+Cypress.Commands.add('importRepo', (remoteUrl, repoName, user) => {
+  cy.createGiteaUser(user).then(giteaUser => {
+    cy.mirrorRepo(remoteUrl, repoName, giteaUser)
+  })
+  cy.waitForStatusCode(
+    `http://gitea.kitspace.test:3000/${user.username}/${repoName}`,
+  )
+})
+
+Cypress.Commands.add('mirrorRepo', (remoteRepo, repoName, user) => {
+  const endpoint = 'http://gitea.kitspace.test:3000/api/v1/repos/migrate'
+  const headers = {
+    'Content-Type': 'application/json',
+    Authorization: `token ${Cypress.env('CYPRESS_GITEA_ADMIN_TOKEN')}`,
```

Comment:
```suggestion
    Authorization: `token ${Cypress.env('GITEA_ADMIN_TOKEN')}`,
```

Reply:
It has to start with `cypress_/CYPRESS_` to be read by cypress automatically  https://docs.cypress.io/guides/guides/environment-variables#Option-3-CYPRESS_